learntools/embeddings/nbs/3-gensim.ipynb

```diff
@@ -129,9 +129,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Our weight matrix has 26,744 rows for that many movies. Each row is 8 numbers - the size of our movie embeddings.\n",
+    "Our weight matrix has 26,744 rows for that many movies. Each row is 32 numbers - the size of our movie embeddings.\n",
     "\n",
-    "Let's look at an example movie vector:"
+    "Let's look at an example movie vector (which happens to be for the movie Toy Story):"
```

Comment:
I kind of prefer the earlier version, despite it being longer. It's sort of another (very minor) instance of problem then solution. I think it makes it easier to understand the code that follows where we cross-reference embeddings with the metadata of their corresponding movies.

Reading this version, my first thought is "how do you know it's *Toy Story*?".